pkg/adapter/receive_adapter.go

```diff
@@ -222,7 +222,7 @@ func sourceFromGitLabEvent(gitLabEvent gitlab.Event, payload interface{}) string
 		return wpe.ObjectAttributes.URL
 	case gitlab.PipelineEvents:
 		pe := payload.(gitlab.PipelineEventPayload)
-		return pe.ObjectAttributes.URL
+		return pe.Project.HTTPURL
```

Comment:
is that going to be the same syntax then before ? e.g `https://gitlab.com/sebgoa/kdemo`

Reply:
Yes, but this value isn't used anymore due to #24.

FYI this is a **backport**, there is no change to be made in this PR.